### PR TITLE
ADD sendScan and endNewRisks

### PR DESCRIPTION
**Pull request by mcpeach9**

### CODE REVIEW

**Review by chjlarson — APPROVED** [review]
Everything seems to be in order! Ran the query in Dev with no issue! Didn't see any syntax errors that would keep it from running. Good Job!

**Review by mcpeach9 — COMMENTED** [review]
@chjlarson made change to the content to reflect sendNewRisks in the title.

**Review by chjlarson — DISMISSED** [review]
Title adjusted to match description and query! Good catch!

**Review by mcpeach9 — COMMENTED** [review]
@austinkelleher , please review and let me know if you would like me to change the alias.
Interested in your feedback

File: jupiterone/questions/questions.yaml [hunk 1]
```diff
@@ -1,4 +1,15 @@
 ---
 sourceId: managed:aquasec
 integrationDefinitionId: "${integration_definition_id}"
-questions: []
+questions: 
+- id: integration-question-Acquasec-user-that-send-security-results-or-sendNewRisks
+  title: Which Acquasec user that sendScanResults or sendNewRisks are set to true?
+  description:
+    Returns all Acquasec users that sendScanResults or sendNewRisks.
```
Comment (zemberdotnet) [inline jupiterone/questions/questions.yaml:8]:
```suggestion
  title: Which Aqua Security user that sendScanResults or sendNewRisks are set to true?
  description:
    Returns all Aqua Security users that sendScanResults or sendNewRisks.
```
Needs one small change to the vendor name.

Comment (mcpeach9) [reply jupiterone/questions/questions.yaml]: Thanks for the clarity, it's appreciated for future reference.

**Review by ericanagle1 — DISMISSED** [review]

**Pull request merged**